debezium-quarkus-outbox/integration-tests/src/test/java/io/debezium/outbox/quarkus/it/OutboxProfiles.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.outbox.quarkus.it;
+
+import java.util.Collections;
+import java.util.Map;
+
+import io.quarkus.test.junit.QuarkusTestProfile;
+
+/**
+ * Specifies a variety of test profiles to be used for the Debezium Outbox extension test suite.
+ *
+ * @author Chris Cranford
+ */
+public class OutboxProfiles {
+    /**
+     * Uses the values directly from {@code application.properties} without changes.
+     */
+    public static class Default implements QuarkusTestProfile {
+    }
+
+    /**
+     * Enables OpenTracing support that is disabled by default.
+     */
```

Comment:
I'm still a bit confused by this. IMO, The rules should be:

* If the OT extension itself is not present, (naturally), there's no tracing support provided by the outbox extension
* If the OT extension itself is present, the outbox tracing support is enabled by default
* If the OT extension itself is present, the outbox tracing support can optionally be disabled by setting `quarkus.debezium-outbox.tracing.enabled` to `false`; 

Is this what we're doing?

Reply:
@gunnarmorling In the confusion, we have done the opposite to the rules said ie. OT is disabled by default and will be enabled via `quarkus.debezium-outbox.tracing.enabled` as `true`.